SPECS/hvloader/hvloader.spec

```diff
@@ -1,17 +1,18 @@
 %define  debug_package %{nil}
```

Comment:
Note - I left off hvloader-signed from the cherry-pick. Happy to add it as well.